### ISSUE TITLE
AutoAugment:Learning Augmentation Strategies from Data

### ISSUE THREAD

**Issue by dltmddbs100** [dltmddbs100/Paper-log#16]
## Main Idea
**Learning and finding efficient augmentation strategies**
Paper: <https://arxiv.org/abs/1805.09501>

## Abstract
Data augmentation is an effective technique for improving the accuracy of modern image classifiers. However, current data augmentation implementations are manually designed. In this paper, we describe a simple procedure called AutoAugment to automatically search for improved data augmentation policies. In our implementation, we have designed a search space where a policy consists of many subpolicies, one of which is randomly chosen for each image in each mini-batch. A sub-policy consists of two operations, each operation being an image processing function such as translation, rotation, or shearing, and the probabilities and magnitudes with which the functions are applied. We use a search algorithm to find the best policy such that
the neural network yields the highest validation accuracy on a target dataset. Our method achieves state-of-the-art accuracy on CIFAR-10, CIFAR-100, SVHN, and ImageNet (without additional data). On ImageNet, we attain a Top-1 accuracy of 83.5% which is 0.4% better than the previous record of 83.1%. On CIFAR-10, we achieve an error rate of 1.5%, which is 0.6% better than the previous state-of-theart. Augmentation policies we find are transferable between datasets. The policy learned on ImageNet transfers well to achieve significant improvements on other datasets, such as Oxford Flowers, Caltech-101, Oxford-IIT Pets, FGVC Aircraft, and Stanford Cars.

## Contribution
+ Data augmentation은 보편성 측면에서 많은 문제를 가짐(타 dataset으로의 전이는 일반화 능력이 떨어짐) => Automatically learned data augmentation을 제안
+ Search Space와 Search Algorithm, The training of controller RNN으로 구성
 1. Search Space는 policy에 해당, 2개의 operation으로 구성된 5개의 sub-policy들의 집합
 2. Search Algorithm이 적합한 augmentation policy를 탐색, 해당 policy를 이용해 network를 학습
 3. 그 과정에서 validation set에 대한 accuracy의 정보가 RNN Controller로 전달되어 update
 4. RNN controller는 validation set으로 부터 얻은 보상에 따라 강화학습, 효과적인 augmentation policy를 탐색하는 방향으로 학습 => operation의 선정과 그 magitude를 최적화
+ AutoAugment를 통한 policy의 선정이 general한 feature들을 잘 표현
+ 타 dataset으로의 augmentation method transfer, 특정 도메인에 적절한 DA들을 찾지만 과적합되지 않는 높은 보편성
+ Best policy의 선정을 위해서는 child model이 적어도 80-120 epochs 동안 학습해야함, 적절한 sub-policy의 수는 20개 이상

## Reviews
<https://deepkerry.tistory.com/27>
